lib/process-services-cloud/src/lib/task/start-task/models/task-details-cloud.model.ts

```diff
@@ -85,6 +85,11 @@ export class TaskDetailsCloudModel {
         return this.status === TaskStatusEnum.ASSIGNED;
     }
 
+    clickAble(): boolean {
```

Comment:
I think the correct word is `clickable`, without capital letter

Reply:
done, and I moved this method to the right place https://github.com/Alfresco/alfresco-ng2-components/pull/5191/files#diff-dc7c734f2a158b34f3f85efaa40a4811R260